bp_be/src/include/bp_be_dcache/bp_be_dcache_pipeline.vh

```diff
@@ -0,0 +1,50 @@
+/**
+ *  Name: 
+ *    bp_be_dcache_pipeline.vh
+ *
+ *  Description: 
+ *    Structs used for dcache pipeline
+ */
+
+`ifndef BP_BE_DCACHE_PIPELINE_VH
+`define BP_BE_DCACHE_PIPELINE_VH
+
+`define declare_bp_be_dcache_size_op_s                                             \
+  typedef struct packed {                                                          \
+    logic double_op;                                                               \
+    logic word_op;                                                                 \
+    logic half_op;                                                                 \
+    logic byte_op;                                                                 \
+  } bp_be_dcache_size_op_s;
+
+`define bp_be_dcache_size_op_width                                                 \
+  (1 + 1 + 1 + 1)
+
+`define declare_bp_be_dcache_pipeline_s                                            \
+  typedef struct packed {                                                          \
+    logic load_op;                                                                 \
+    logic store_op;                                                                \
+    logic signed_op;                                                               \
+    logic lr_op;                                                                   \
+    logic sc_op;                                                                   \
+    logic amoswap_op;                                                              \
+    logic amoadd_op;                                                               \
+    logic amoxor_op;                                                               \
+    logic amoand_op;                                                               \
+    logic amoor_op;                                                                \
+    logic amomin_op;                                                               \
+    logic amomax_op;                                                               \
+    logic amominu_op;                                                              \
+    logic amomaxu_op;                                                              \
+    bp_be_dcache_size_op_s size;                                                   \
+    logic fencei_op;                                                               \
```

Comment:
Add field "l2_op".  This will be decoded in the decoder and used to decide whether to send reqs / return v_o, rather than checking parameters there